### PR TITLE
update installation instructions

### PR DESCRIPTION
**Pull request by paregorios**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -22,11 +22,15 @@ Pleiades package distributions are fetched from http://atlantides.org/eggcarton/
...
 
-1. Clone the repo.
-2. Run "virtualenv-2.7 --no-setuptools ." in the repo directory.
-3. Run "bin/python bootstrap.py"
-4. Run "bin/buildout -c devel.cfg"
-5. Start Zope instance in foreground with "bin/instance fg"
-6. Go to http://localhost:9080/manage
-7. Click Add Plone Site and create a Plone site named `plone`
+1. Install prerequisites (TBD: add list, eg libgeos)
+2. Clone the repos
```
Comment (davisagli) [inline README.rst:26]: Just this one; the mr.developer extension for buildout will fetch the others.

File: README.rst [hunk 2]
```diff
@@ -22,11 +22,15 @@ Pleiades package distributions are fetched from http://atlantides.org/eggcarton/
...
+3. Create a Python 2.7 virtual environment and activate it
+4. In the clone, checkout the jazkarta-plone4 branch
```
Comment (davisagli) [inline README.rst:28]: Any objection to renaming branches so master becomes something for achival purposes and `jazkarta-plone4` becomes master?

File: README.rst [hunk 3]
```diff
@@ -22,11 +22,15 @@ Pleiades package distributions are fetched from http://atlantides.org/eggcarton/
...
+5. Run "bin/python bootstrap.py"
+6. Run "bin/buildout -c devel.cfg"
+7. If you need to create an admin login, do: bin/instance adduser <name> <password>
```
Comment (davisagli) [inline README.rst]: This should not be necessary for a fresh install; an admin user with password admin is created by default.

File: README.rst (hunk 2 above)
Comment (paregorios) [reply README.rst:28]: I would very much like to see that change occur.

**Pull request merged**